### ISSUE TITLE
Review EFO terms added for anatomograms

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#880]
As per conversation with @dosumis and @zoependlington :
(NOTE, the strategy below is superseded by the one [here](https://github.com/EBISPOT/efo/issues/880#issuecomment-795737707))

- [x] Review terms added to EFO for the anatomograms so far, to check if there's content we should migrate to CL/Uberon. 
- [x] Obsolete/replace the EFO terms with CL/Uberon ones
- [x] Heads up to the SCEA team when all is done.

Based on the anatomograms in https://github.com/EBISPOT/efo/tree/master/src/templates, the following EFO terms were created as placeholders for CL/Uberon terms and should be added there instead:

'airway submucosal gland' <http://www.ebi.ac.uk/efo/EFO_0010666> 
(now https://github.com/obophenotype/uberon/issues/1786)
(UPDATE 4/6/21: this is now UBERON:8410043 bronchus submucosal gland (synonym: "airway submucosal gland"), with an xref to EFO:0010666 'airway submucosal gland'.)

'central venous hepatic sinusoidal endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010706>
'endometrial glandular epithelial cell' <http://www.ebi.ac.uk/efo/EFO_0010710>
'endothelial cell of respiratory system lymphatic vessel' <http://www.ebi.ac.uk/efo/EFO_0010667>
'fused extravillous trophoblast' <http://www.ebi.ac.uk/efo/EFO_0010712>
'lung capillary endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010668>
'lung pericyte cell' <http://www.ebi.ac.uk/efo/EFO_0010669>
'periportal hepatic sinusoidal endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010705>
'placental endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010708>
'placental smooth muscle cell' <http://www.ebi.ac.uk/efo/EFO_0010709>
'portal endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010704>
'uterine endothelial cell' <http://www.ebi.ac.uk/efo/EFO_0010711>
(now https://github.com/obophenotype/cell-ontology/issues/932)

**Comment by paolaroncaglia** [comment]
As per discussion at CL developers' meeting and with Single Cell Atlas, the EFO terms created in #890 should be replaced by CL ones:

- [ ] Create CL terms as resolved at CL meeting (subset etc.)
- [ ] Obsolete the corresponding EFO terms and replace them with CL ones
- [ ] Check that Single Cell Atlas pipeline is ok with replacements: @dosumis @zoependlington @ngeorgeebi could you confirm please?

Thanks,
Paola

**Comment by paolaroncaglia** [comment]
A procedural note: at the curators call today we agreed on a provisional plan:
Phase 1: Add to Uberon/CL and link via xrefs to EFO
Warn SCA of pending obsoletion (some months)
Phase2: Obsolete in EFO and add replaced_by tag pointing to Uberon/CL term.

**Comment by paolaroncaglia** [comment]
@sfexova @ngeorgeebi cc @dosumis and @zoependlington 

Following up on the plan [above](https://github.com/EBISPOT/efo/issues/880#issuecomment-795737707), I'd like to officially start the EFO terms obsoletion notice. How long do you think your team would need before we can go ahead, please? Is there anyone else we should bring into this conversation here, or is that something you could discuss yourselves and let us know? If you have any concern or question, please feel free to say. The correspondence between (old) EFO terms and (new) CL + Uberon terms is [here](https://docs.google.com/spreadsheets/d/1Ao9d2VR4xqQsesHDz6Dn27V4ySfGntd-tbaUVeZ-NYI/edit#gid=0). All new terms have an xref pointing to corresponding old terms. All old term names (labels), when different from new ones, are present as synonyms.

Thanks,
Paola

**Comment by dosumis** [comment]
@zoependlington - are you now happy to obsolete these?

**Comment by zoependlington** [comment]
In that case I will hold off implementing for this upcoming release, thanks for the update @sfexova!

**Comment by sfexova** [comment]
Hi @zoependlington - Can I just ask how exactly does the obsoletion process work? Do you first deprecate/hide the terms in EFO and then after a while (how long?) jettison them entirely or are they dropped completely once the obsoletion goes ahead or do they stay there forever but hidden?

**Comment by zoependlington** [comment]
@sfexova Furthering what David has said, these terms would remain in EFO but lose all of their relationships to other terms. Every obsoleted term (unless in extreme circumstances) has a replaced by term using a standard annotation (as explained in David's comment above) which is used by the likes of Open Targets for example to automatically update any IDs they are using that may be obsoleted and replaced by other terms. Effectively the replacement term would take the place of the current term and the current term would be retired forever, meaning still existing in EFO but outside of the hierarchy entirely and only used as a link to the new term. I hope this makes things clearer!

**Comment by paolaroncaglia** [comment]
Note to self, I added this ticket for discussion at the next monthly single-cell curators call, to ensure follow-up.

**Comment by paolaroncaglia** [comment]
@sfexova , hi, thank you for the update, that's all right. Could you please give an estimate of when the next data release would be? So we have an idea and don't keep pestering you if it's too soon ;-) Just a ballpark (2 months, 4 months...). Thank you.

**Comment by paolaroncaglia** [comment]
Update from today's meeting: the data release could easily take 3 months. So if we don't hear from SCA sooner, we'll ask for an update around mid-June.

**Comment by paolaroncaglia** [comment]
@zoependlington (see earlier comment) I'll remove my assignment then, but do let me know if you need me to do/clarify anything. Thank you.

**Comment by sfexova** [comment]
Hi all, sorry to be re-opening this painful subject again, but can we have another look at the 'airway submucosal gland' being replaced with 'bronchus submucosal gland' I think we will need a broader term than bronchus submucosal gland here, the SMGs are apparently located along the whole respiratory tract, including the nose

**Comment by paolaroncaglia** [comment]
@sfexova 

> Hi all, sorry to be re-opening this painful subject again, but can we have another look at the 'airway submucosal gland' being replaced with 'bronchus submucosal gland' I think we will need a broader term than bronchus submucosal gland here, the SMGs are apparently located along the whole respiratory tract, including the nose

The [EFO term created originally](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0010666) has a broad def "Gland in the submucosal walls of the airways." but a specific location 
Subclass of: 
[gland](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http://purl.obolibrary.org/obo/UBERON_0002530)
[part of](https://www.ebi.ac.uk/ols/ontologies/efo/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050) some [bronchus](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002185)
so that's where 'bronchus submucosal gland' comes from (see https://github.com/obophenotype/uberon/issues/1786).
Anyway,
- [ ] I'll create a new Uberon term for 'airway submucosal gland' as a child of 'submucosal gland' (and parent of 'bronchus submucosal gland').

**Comment by paolaroncaglia** [comment]
@sfexova , hi, as per your request above, I created UBERON:8410077 airway submucosal gland. It will be available at the next Uberon release, scheduled for tomorrow if all goes well (Uberon releases are not super-easy, hence not done too frequently at the moment, but that's the plan). Cheers.

**Comment by aleixpuigb** [comment]
Hi @sfexova, I wanted to revisit this issue since all terms are already available in Uberon and CL. I wanted to check if there has been any progress on your side, so we can start the obsoletion process for these terms. Thanks!

The original list of 13 EFO terms have already been created in Uberon or CL:

| EFO label                                                       | EFO IRI                                                | CL/Uberon label                                      | CL/Uberon IRI                                               |
|-----------------------------------------------------------------|--------------------------------------------------------|------------------------------------------------------|-------------------------------------------------------------|
| airway submucosal gland                                         | http://www.ebi.ac.uk/efo/EFO_0010666                  | airway submucosal gland                              | http://purl.obolibrary.org/obo/UBERON_8410077              |
| central venous hepatic sinusoidal endothelial cell              | http://www.ebi.ac.uk/efo/EFO_0010706                   | endothelial cell of pericentral hepatic sinusoid     | http://purl.obolibrary.org/obo/CL_0019022                  |
| endometrial glandular epithelial cell                           | http://www.ebi.ac.uk/efo/EFO_0010710                   | glandular epithelial cell of endometrium             | http://purl.obolibrary.org/obo/CL_0009084                  |
| endothelial cell of respiratory system lymphatic vessel         | http://www.ebi.ac.uk/efo/EFO_0010667                   | endothelial cell of respiratory system lymphatic vessel | http://purl.obolibrary.org/obo/CL_0009086               |
| fused extravillous trophoblast                                  | http://www.ebi.ac.uk/efo/EFO_0010712                   | fused extravillous trophoblast                        | http://purl.obolibrary.org/obo/CL_0009087                  |
| lung capillary endothelial cell                                 | http://www.ebi.ac.uk/efo/EFO_0010668                   | pulmonary capillary endothelial cell                  | http://purl.obolibrary.org/obo/CL_4028001                  |
| lung pericyte cell                                              | http://www.ebi.ac.uk/efo/EFO_0010669                   | lung pericyte                                        | http://purl.obolibrary.org/obo/CL_0009089                  |
| periportal hepatic sinusoidal endothelial cell                  | http://www.ebi.ac.uk/efo/EFO_0010705                   | endothelial cell of periportal hepatic sinusoid      | http://purl.obolibrary.org/obo/CL_0019021                  |
| placental endothelial cell                                      | http://www.ebi.ac.uk/efo/EFO_0010708                   | endothelial cell of placenta                          | http://purl.obolibrary.org/obo/CL_0009092                  |
| placental smooth muscle cell                                    | http://www.ebi.ac.uk/efo/EFO_0010709                   | smooth muscle cell of placenta                        | http://purl.obolibrary.org/obo/CL_0009093                  |
| portal endothelial cell                                         | http://www.ebi.ac.uk/efo/EFO_0010704                   | endothelial cell of hepatic portal vein               | http://purl.obolibrary.org/obo/CL_0009094                  |
| uterine endothelial cell                                       | http://www.ebi.ac.uk/efo/EFO_0010711                   | endothelial cell of uterus                            | http://purl.obolibrary.org/obo/CL_0009095                  |

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26)** of this long-running backlog issue.

**The ask (2020):** 12 EFO terms created as placeholders for the Expression Atlas anatomograms should be replaced by CL/UBERON terms and obsoleted in EFO (agreed plan: create upstream with xrefs → warn SCA → obsolete in EFO with `replaced_by`).

**Where things stand:**

- **Upstream is done, and has been for years.** obophenotype/cell-ontology#932 closed 2022-02-10, obophenotype/uberon#1786 closed 2022-03-24. All 12 replacements exist in OLS and are non-obsolete, and 11 of 12 carry an xref pointing back at the exact EFO ID they replace.
- **Nothing has happened in EFO.** All 12 terms are still active in `efo-edit.owl` — no `owl:deprecated`, no `IAO:0100001`, still under their CL/UBERON parents, still carrying the original `"Made as a placeholder for AE anatomagrams, term should be requested in CL/UBERON."` comment. They are also still referenced in `src/templates/anatomagram_{lung,placenta,liver}.tsv`, and none of the replacement CL/UBERON IRIs are in `iri_dependencies/` yet.
- **The last blocker was on the SCEA side**, pending a data release. @aleixpuigb re-pinged in July 2025 with no reply since.

Proposed mapping, for the record:

| EFO term | Replacement |
|---|---|
| EFO:0010666 airway submucosal gland | UBERON:8410077 |
| EFO:0010667 endothelial cell of respiratory system lymphatic vessel | CL:0009086 |
| EFO:0010668 lung capillary endothelial cell | CL:4028001 pulmonary capillary endothelial cell — **needs confirmation, no xref** |
| EFO:0010669 lung pericyte cell | CL:0009089 |
| EFO:0010704 portal endothelial cell | CL:0009094 |
| EFO:0010705 periportal hepatic sinusoidal endothelial cell | CL:0019021 |
| EFO:0010706 central venous hepatic sinusoidal endothelial cell | CL:0019022 |
| EFO:0010708 placental endothelial cell | CL:0009092 |
| EFO:0010709 placental smooth muscle cell | CL:0009093 |
| EFO:0010710 endometrial glandular epithelial cell | CL:0009084 |
| EFO:0010711 uterine endothelial cell | CL:0009095 |
| EFO:0010712 fused extravillous trophoblast | CL:0009087 |

**One loose end worth flagging:** the sibling batch from #890 was obsoleted in 3.55.0 (EFO:0010894–0010897 → CL:0009021/0022/0023/0024), but **EFO:0010898 'mesothelial cell of large intestine' was missed** — I confirmed it is still active, still carrying a live `equivalentClass` axiom, while all four siblings are deprecated. #1201 requested exactly that obsoletion and was closed with "this task is now part of #880", so it appears to have slipped. Its replacement CL:0009025 (now labelled 'mesothelial cell of colon') exists and is fine to use.

**Recommendation:** keep open — this is unblocked on the ontology side and mechanical to finish (import the 12 replacements, obsolete with `replaced_by`, update the anatomogram TSVs, plus EFO:0010898). @sfexova / Expression Atlas team — is the SCEA data migration off these EFO IDs now complete? Given the replacements have been available since 2022 and `replaced_by` keeps the old IDs resolvable, it may be reasonable to proceed absent an objection. Please also confirm CL:4028001 for EFO:0010668, the one mapping without a confirming xref.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
